### ISSUE TITLE
FR: explain required permissions on macOS

### ISSUE THREAD

**Issue by dandv** [mucommander/mucommander#924]
After installing muCommander on Ventura 13.3.1, when starting the application, I see this message:

> “muCommander.app” wants access to control “System Events.app”. Allowing control will provide access to documents and data in “System Events.app”, and to perform actions within that app.

<img width="263" alt="image" src="https://github.com/mucommander/mucommander/assets/33569/050cc6d4-19ba-47af-9ff2-edf1aa4a1c24">

It would help to tell the user why that permission is necessary, before requesting it.

**Comment by dandv** [comment]
Another one, after deleting a file:

> “muCommander.app” wants access to control “Finder.app”. Allowing control will provide access to documents and data in “Finder.app”, and to perform actions within that app.

It's a bit unclear why this is required?

**Comment by ahadas** [comment]
@dandv AFAIK these operations are produced by macOS when it detects the application tries to do something that requires those permissions, e.g., if you try to browse the 'Documents' folder it will produce that error that asks you to grant the right permission for the application to be able to do that, so I'm afraid there is no way to provide a more detailed information that what you see in these windows but please let me know if you get better messages by other applications and I'll try to see how they do that differently.

Specifically about the cases you've mentioned - we use finder to run several scripts, e.g., to get the content of the Trash. We use system events to reflect that actions, like file copying, finish.

**Comment by dandv** [comment]
> these messages are produced by macOS when it detects the application tries to do something that requires those permissions, e.g., if you try to browse the 'Documents' folder it will produce that message that asks you to grant the right permission for the application to be able to do that

That's correct, and also not the case here, so maybe something else can be done.

In the case of the "System Events.app" app, I didn't do anything specific with muCommander, other than starting it. What I'm suggesting is displaying to the macOS user a dialog as early as possible, warning that macOS will show these system dialogs about System Events and Finder, and that they're expected and should be allowed (or not?). My concern is tha some more paranoid users might thing muCommander is attempted to perform a malicious action, or that the download got infected etc.
